cmd/ddev/cmd/commands.go

```diff
@@ -103,9 +103,12 @@ func addCustomCommands(rootCmd *cobra.Command) error {
 				var flags Flags
 				flags.Init(commandName, onHostFullPath)
 
+				disableFlags := true
 				if val, ok := directives["Flags"]; ok {
 					if err = flags.LoadFromJSON(val); err != nil {
 						util.Warning("Error '%s', command '%s' contains an invalid flags definition '%s', skipping add flags of %s", err, commandName, val, onHostFullPath)
+					} else {
+						disableFlags = false
 					}
```

Comment:
```suggestion
					disableFlags = false
					if err = flags.LoadFromJSON(val); err != nil {
						util.Warning("Error '%s', command '%s' contains an invalid flags definition '%s', skipping add flags of %s", err, commandName, val, onHostFullPath)
					}
```

I would prefer always to set `disableFlags` to false if flags are present. No need they could be properly loaded I'd say.

Reply:
@gilbertsoft ok, I did this in a slightly different way.